tests/integration/security/authorization_test.go

```diff
@@ -1066,3 +1069,56 @@ func TestAuthorization_Path(t *testing.T) {
 			rbacUtil.RunRBACTest(t, cases)
 		})
 }
+
+// //Test that LOG action does not impact authorization
```

Comment:
?

Reply:
This is a commented out integration test that should work with a proxy update.

Reply:
This PR is still WIP for that reason